src/components/ProfileButton.vue

```diff
@@ -11,13 +11,12 @@
     </q-btn>
 
     <li v-if="auth.authenticated" class="menu-btn" v-on:click="open = !open">
-      <span>{{ auth.firstname }} {{ auth.lastname }}</span>
+      <span>{{ auth.firstname }} {{ auth.lastname }}</span> &nbsp;&nbsp;
       <q-icon
         :name="open ? 'fas fa-chevron-up' : 'fas fa-chevron-down'"
       ></q-icon>
       <ul :class="open ? '' : 'hidden'">
         <li v-on:click="$router.push('/profile')">PROFIL</li>
-        <li v-on:click="$router.push('/profile/requests')">MEINE ANZEIGEN</li>
```

Comment:
Not needed anymore?

Reply:
The Myrequests page is now available in a tab in the Profile page.